### PR TITLE
Create eve_market.md

### PR DESCRIPTION
**Pull request by bahrmichael**
Added a documentation for the market endpoint (explaining history). Hope this suits the doc standards.

### CODE REVIEW

**Comment by JimiC** [comment]
You may wonna add your name in CONTRIBUTORS too. Give credits to yourself.

File: docs/crest/eve/eve_market.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Market
+The market resource allows an application to read market data.
+
+## History
+### Route
+``/market/{regionId}/history/{type-href}``
```
Comment (Regner) [inline docs/crest/eve/eve_market.md]: Query parameters should go into a Parameters section like the XML API. Shown like this it looks like you don't know that it is a parameter.

Comment (Regner) [inline docs/crest/eve/eve_market.md]: Another part of that is that query parameters are different per version of the resource and also per verb. You don't need the same query parameters when POSTing for example.

Comment (bahrmichael) [inline docs/crest/eve/eve_market.md]: Thanks for the heads up! Two questions:

1. For the parameter section, I found [something](https://github.com/ccpgames/eveonline-third-party-documentation/blob/master/docs/xmlapi/eve/eve_alliancelist.md) I can follow, but the [other](https://github.com/ccpgames/eveonline-third-party-documentation/blob/master/docs/crest/eve/eve_opportunities.md) crest APIs don't use that format either. I am unsure what the correct way is. Can you give me an example?

2. Is there an example for describing the `href`-part (another crest url)? As far as I know this is not possible for XML.

**Comment by bahrmichael** [comment]
Added info from https://developers.eveonline.com/blog/article/new-crest-resource-for-bulk-market-orders, but still unsure about the parameter handling.

**Comment by Regner** [comment]
Will try and respond tonight. Last week at the office and a lot to do so sorry for the delayed responses.

**Comment by Regner** [comment]
For the parameters I think it would be best just to have Paramaters section under the Route section (still H4) that has a l list of possible parameters for the route.

Does anyone see a better way to handle params?

**Comment by bahrmichael** [comment]
How can we get more insight on the parameters? Which ones are available, what values do they expect, what changes when I use different verbs?

**Comment by deadlybulb** [comment]
You typically get this data by making an OPTIONS request, although it's not always complete in my experience.  Good overview of this here: https://www.youtube.com/watch?v=QMQOjUjrZIo

**Comment by bahrmichael** [comment]
The OPTIONS request explains what content is delivered from the server, but does not give insight on the request parameters one can use. So the request parameters are still unclear.

**Comment by polarina** [comment]
Thanks! I've merged your pull request.

**Pull request closed without merging**